pkgs/misc/uq/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "uq";
+  version = "2018-05-27";
```

Comment:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 

Alternatively you could ask upstream for a release.

Reply:
I've had a chat in the past with the developer, he's not putting any more effort to this program but I think it's still usable / useful.